libmachine/engine/engine.go

```diff
@@ -1,5 +1,9 @@
 package engine
 
+const (
+	DefaultPort = 2376
```

Comment:
This was pretty long overdue anyway so I've gone ahead and started pulling instances of this magic variable into a constant.